apps/service_profile/lib/profile/event/handler.ex

```diff
@@ -0,0 +1,14 @@
+defmodule Profile.Event.Handler do
+  use Brook.Event.Handler
+
+  import Events, only: [extract_start: 0, profile_update: 0]
+
+  def handle_event(%Brook.Event{type: extract_start(), data: %Extract{} = extract}) do
+    Profile.Feed.Supervisor.start_child(extract)
```

Comment:
do we handle an extract_end yet? how are we cleaning up a feed without a corollary "end event"?

Reply:
An extract_end event can happen alot with a dataset that is ran on cron.   To my knowledge we haven't sat down and determined how we will know when a dataset is no longer being ingested.

Reply:
so is the plan for this supervisor child to just sit there and wait until more data arrives?

Reply:
I believe so, isn't that the way broadcast and persist work? or has there been a change to those?

Reply:
they both handle `persist:end` events to clean up their children; nothing about `extract:end` but then they don't respond to `extract:start` to begin with, just the `broadcast/load:persist:start/end`

Reply:
extract:end doesn't make sense, but maybe a profile:end could be created, but like persist:end and broadcast:end, we don't have anyway to know when to send them.

Reply:
feels safer to have it so we don't have this one service failing to clean up after its children when we do institute end event handling. if all the other apps already handle those events that makes missing one more likely imo

Reply:
my vote would be we groom stories about how we are going to clean up datasets in an automated fashion.  persist:end and broadcast:end are coded, but I am not sure that is even useful since nothing sends them and I am not sure we want something to track which loads have been sent for which dataset.  We might want a more general dataset is done being ingested event.